components/asana/sources/new-tag/new-tag.mjs

```diff
@@ -0,0 +1,50 @@
+import asana from "../../asana.app.mjs";
+
+export default {
+  key: "asana-new-tag",
+  type: "source",
+  name: "New Tag",
+  description: "Emit new event for each tag created in a workspace.",
+  version: "0.1.0",
```

Comment:
From this previous comment - https://github.com/PipedreamHQ/pipedream/pull/2487#discussion_r842896536
```suggestion
  version: "0.0.1",
```